packages/react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -282,7 +282,7 @@ export default class Tooltip extends AutoControlledComponent<TooltipProps, Toolt
       ...this.getContentProps(),
     }
 
-    const tooltipContent = Tooltip.Content.create(content, {
+    const tooltipContent = Tooltip.Content.create(content || '', {
```

Comment:
When `content` is null, the `.create()` factory here will result in `tooltipContent` being `null` as well.  It seems the page breaks because the `<Ref>` on line 297 requires a single React child.

Instead of inlining a default value of `''` when `content` doesn't exist, this whole `renderPopperChildren()` method should return null if there is no `tooltipContent`.  Something like:

```diff
const tooltipContent = Tooltip.Content.create(...)

+if (!tooltipContent) {
+  return null
+}

return <Ref innerRef={this.contentRef}>{tooltipContent}</Ref>
```

This solves a couple issues with the current PR:
- Default props, if required, should be added in `defaultProps` opposed to inline.
- If there is no content, we shouldn't create an empty content element with a ref, but just render null instead.

Reply:
Thank you for review, it will fixed in 2-3 days according to your comment

Reply:
Changed. However, in order to return null, I had to change also Popper.tsx to return null when content is empty as Ref doesn't accept null children